app/scripts/sidepanel-helper.ts

```diff
@@ -0,0 +1,87 @@
+import browser from 'webextension-polyfill';
+import { CONTENT_SCRIPT } from './constants/stream';
+import { EXTENSION_MESSAGES } from '../../shared/constants/messages';
+
+const isSidepanelEnabled = process.env.IS_SIDEPANEL === 'true';
+
+const METHODS_REQUIRING_UI = [
```

Comment:
This seems like a flawed way to do this? We will have to remember to add new methods, some existing ones haven't even made it to this list. Some methods may require UI _sometimes_.

Can't we open the side panel based on an approval being created, similarly to how the popup works?

Reply:
Yeah not a fan of maintaining a list either

Will look at options